package.json

```diff
@@ -17,9 +17,13 @@
     "url": "https://github.com/orgs/plantemoran-ai-engineering/.github/issues"
   },
   "scripts": {
-    "lint": "markdownlint **/*.md"
+    "format": "prettier . --write --log-level log",
+    "format:check": "prettier . --check",
+    "lint": "markdownlint **/*.md",
+    "lint:md": "markdownlint **/*.md"
```

Comment:
You don't need this.
Your `lint` script only does 1 thing (lint MD) and uses the same command.
```suggestion
```